app/assets/javascripts/dashboard.js

```diff
@@ -111,7 +111,6 @@ app.controller('MapController',[ '$scope','$http' ,function($scope,$http) {
         google.maps.event.addListener(map, 'click', function(event) {
```

Comment:
It might make sense to define this callback elsewhere; it'd help keep your code cleaner/easier to read